targets/hasura/migrations/default/1723964804960_alter_table_information_informations_add_column_display_date/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+-- alter table "information"."informations" add column "display_date" date
+--  null;
```

Comment:
il faudrait mettre un down ici vu que le up alter le schéma

Reply:
ouaip, mais bon, on l'a fait nul part.... puis après faut le tester, etc. est-ce worth it ?